receiver/receiver.go

```diff
@@ -80,6 +80,16 @@ func ReadTimeout(seconds uint32) Option {
 	}
 }
 
+// ConcatChar creates option for New contructor
```

Comment:
Typo in `constructor`?

Reply:
Good catch, I just copied from the one above 😄